src/index.html

```diff
@@ -1,10 +1,12 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-    <meta charset="UTF-8">
-    <title>React scaffold by ngParty!!!</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="shortcut icon" href="favicon.ico">
```

Comment:
don't get what exactly is this line solving when all major browsers try to download favicon file automatically, so this doesn't have much sense, also favicon.ico file is missing in this project 😉